form.yml

```diff
@@ -0,0 +1,329 @@
+    $type: boolean
+    $default: false
+    $help: Enable the node monitoring via exporters which will be installed and configured in all the nodes.
+
+  nodes:
+    $name: Nodes
+    $type: edit-group
+    $minItems: 1
+    $itemName: ""
+    $prototype:
+      host:
+        $name: Hostname to install Netweaver
+        $type: text
+        $optional: false
+      virtual_host:
+        $name: Virtual Hostname
+        $type: text
+        $optional: false
+      ascs_virtual_host_checkbox:
+        $name: Provide the ASCS Virtual Hostname (needed for PAS instance only)
+        $type: boolean
+        $default: false
+        $help: If a nfs share is used to manage the HA mounting point
+      ascs_virtual_host:
+        $name: ASCS Virtual Hostname
+        $visibleIf: .ascs_virtual_host_checkbox == true
+        $type: text
+        $optional: true
+      virtual_host_interface:
+        $name: Virtual host interface
+        $type: text
+        $default: eth0
+        $optional: true
+      virtual_host_mask:
+        $name: Virtual host mask
+        $type: text
+        $default: 32
+        $optional: true
+      sid:
+        $name: SAP system identifier (SID)
+        $type: text
+        $optional: false
+        $help: System ID of the HANA installation
+      instance:
+        $name: SAP instance number
+        $type: text
+        $optional: false
+      root_user:
+        $name: Root user
+        $type: text
+        $optional: false
+      root_password:
+        $name: Root user password
+        $type: password
+        $optional: false      
+      shared_disk_dev:
+        $name: Shared disk device (valid only for libvirt provider)
+        $type: text
+        $optional: true
+        $help: Set the shared disk used in HA environments. Skip this parameter in non HA environments
+      use_nfs_disk_checkbox:
+        $name: Use nfs share SID folder
+        $type: boolean
+        $default: false
+        $help: If a nfs share is used to manage the HA mounting point
+      init_shared_disk:
+        $name: Init the shared disk
+        $visibleIf: .use_nfs_disk_checkbox == true
+        $type: boolean
+        $optional: true
+        $help: Init the shared disk. Only used if a shared disk is provided
+      product_id:
+        $name: SAP product ID
+        $type: text
+        $help: Set an specific product id. This will have the preference over global product id
+        $optional: true      
+      sap_instance:
+        $name: SAP Instance
+        $type: text
+        $help: Provide the netweaver instance/server type - ascs, ers, db, pas, aas
+        $optional: false
+      extra_parameters_checkbox:
+        $name: Provide extra installation parameters
+        $type: boolean
+        $optional: true  
+      extra_parameters:
+        $name: Installation extra options
+        $visibleIf: .extra_parameters_checkbox == true
+        $help: Optional configuration parameters
```

Comment:
Please, improve the help. The extra parameters are additional `key/value` entries that will be updated in the used installation configuration file